federatedscope/mf/dataset/movielens.py

```diff
@@ -18,16 +18,20 @@ class VMFDataset:
 
     """
     def _split_n_clients_rating(self, ratings: csc_matrix, num_client: int,
-                                test_portion: float):
+                                split: list):
```

Comment:
How about enabling this change to FedNetflix?

Reply:
FedNetflix is inherited from MovieLensData, thus this change should be valid to FedNetflix